### DISCUSSION TITLE
Loading hook in vim-airline

### DISCUSSION THREAD

**Discussion by princejoogie** [vuki656/package-info.nvim#118] [Q&A] [answered]
### Issues

- [X] I have checked [existing issues](https://github.com/vuki656/package-info.nvim/issues?q=is%3Aissue) and there are no existing ones with the same request.

### Feature description

how to i show status with loading hook in [vim-airline](https://github.com/vim-airline/vim-airline)?

### Help

No, sorry.

**Comment by princejoogie** [comment]
also how do i only bump the version in package,json file but not run `yarn/npm install`?

**Comment by vuki656** [comment]
> ### Issues
> 
> - [X] I have checked [existing issues](https://github.com/vuki656/package-info.nvim/issues?q=is%3Aissue) and there are no existing ones with the same request.
> 
> ### Feature description
> 
> how to i show status with loading hook in [vim-airline](https://github.com/vim-airline/vim-airline)?
> 
> ### Help
> 
> No, sorry.
> 
> ### Implementation help
> 
> _No response_

Can you share your current status line config?

Different status lines accept custom sections in different ways so I'd need the example to see how we can do it.

**Comment by vuki656** [comment]
> also how do i only bump the version in package,json file but not run `yarn/npm install`?

Unfortunately you can't since the commands run `npm i pgk` under the hood and those automatically install the dependency as well.

**Comment by princejoogie** [comment, marked as answer]
ditched vim-airline for [lualine](https://github.com/nvim-lualine/lualine.nvim) which i got the hooks working with lua

![image](https://user-images.githubusercontent.com/47204120/159092808-63a49e4f-7f14-4423-93db-725b01926824.png)

![image](https://user-images.githubusercontent.com/47204120/159093078-e5d6b17e-e67c-4dd0-8a5d-a664a1fcfb83.png)
